features/old/services/subscriptions/bulk_operations/send_emails.feature

```diff
@@ -25,34 +25,27 @@ Feature: Mass email bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
     Given I am logged in as provider "foo.3scale.localhost"
 
-  Scenario: Emails can be sent without body
+  Scenario: Emails can't be sent without body
     Given provider "foo.3scale.localhost" has "service_plans" visible
     And I am on the service contracts admin page
     And a clear email queue
     When I check select for "jane"
     And I press "Send email"
     And I fill in "Subject" with "Nothing to say"
     And I fill in "Body" with ""
-    And I press "Send" and I confirm dialog box within colorbox
-    Then I should see "Action completed successfully"
-    And "jane@me.us" should receive an email with the following body:
-    """
-    """
+    And I press "Send"
+    Then I should see "Selected Service Subscriptions"
```

Comment:
Could we assert instead `And "jane@me.us" should not receive an email with the following body:`? The test has turned a bit vague.

Reply:
This goes also for the previous PR.

Reply:
@josemigallas  I have updated test case

Now test is clear i think,

```
And I press "Send"
Then I should see "Selected Service Subscriptions"
  And a clear email queue
```

These line explains that we after pressing send button we still remain on same page where subscriptions are selected and email queue is also clear which tends to no email shoot.

Also if I say about the UI part when user don't enter body and subject now they are required filed so new action will be trigerred.